Index: internal/adapter/handler/websocket/message.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package websocket\n\nimport \"encoding/json\"\n\n// Event represents a valid websocket event.\ntype Event string\n\nconst (\n\tEventError Event = \"error\"\n\n\tEventAddSession   Event = \"add_session\"\n\tEventSessionAdded Event = \"session_added\"\n\n\tEventUpdateSession  Event = \"update_session\"\n\tEventSessionUpdated Event = \"session_updated\"\n\n\tEventAddOrderedProduct   Event = \"add_ordered_product\"\n\tEventOrderedProductAdded Event = \"ordered_product_added\"\n)\n\n// Message represent a websocket message.\ntype Message struct {\n\tEvent Event           `json:\"event\"`\n\tData  json.RawMessage `json:\"data\"`\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/internal/adapter/handler/websocket/message.go b/internal/adapter/handler/websocket/message.go
--- a/internal/adapter/handler/websocket/message.go	(revision d549eae8a5cecb4212187716d6b4bc69d8eeab7d)
+++ b/internal/adapter/handler/websocket/message.go	(date 1776616130136)
@@ -16,6 +16,9 @@
 
 	EventAddOrderedProduct   Event = "add_ordered_product"
 	EventOrderedProductAdded Event = "ordered_product_added"
+
+	EventDeleteOrderedProduct  Event = "delete_ordered_product"
+	EventOrderedProductDeleted Event = "ordered_product_deleted"
 )
 
 // Message represent a websocket message.
